cmd/gnodev/README.md

```diff
@@ -0,0 +1,23 @@
+# gnodev
+
+`gnodev` is a tool for managing Gno source code.
+
+## Usage
+
+`gnodev <command> [arguments]`
+
+## Commands
+
+* `gnodev run` - run a Gno file
+* `gnodev build` - build a gno package
+* `gnodev precompile` - precompile .gno to .go
+* `gnodev test` - test a gno package
+* `gnodev repl` start a GnoVM REPL
+
+## Install
```

Comment:
I would probably also mention we have a `Makefile` shorthand:

`make install gnodev`

Reply:
Yeah, I’ll update, thanks.

Btw, it’s `make install_gnodev`; `make install gnodev` is the equivalent of `make install && make gnodev`.

Reply:
Done in [53f2148](https://github.com/gnolang/gno/pull/505/commits/53f21486f0b545aa30847e2dec37564744e097a9)